provider/puppet.rb

```diff
@@ -439,35 +444,16 @@ def emit_resourceref_object(data)
       }
     end
 
-    def property_map
```

Comment:
This whole thing was so verbose. Maybe someday we'll need this level of detail, but I feel like we don't need it today.